### DISCUSSION TITLE
Wazuh woes

### DISCUSSION THREAD

**Discussion by Yoyu88** [Security-Onion-Solutions/securityonion#6077] [Unsupported Versions] [answered]
Guys, i have a lab running with SO and that is working fine, the issue is Wazuh.  I am using the MSI installer within PDQ deploy to agents in my test lab.  Using the Following syntax.  There is no password or anything on for registration.  I have the right creds running.   What am I missing here?

wazuh-agent-3.13.3-1.msi /q WAZUH_MANAGER="10.0.0.2" WAZUH_REGISTRATION_SERVER="10.0.0.2"  

WAZUH_REGISTRATION_SERVER="10.0.0.2"

FIGURED IT OUT!!!

**Comment by Yoyu88** [comment, marked as answer]
Figured it out guys

**Reply by DinhDiesel** [reply]
nice work man!
